src/components/cells/markdown-cell.jsx

```diff
@@ -52,14 +44,6 @@ export class MarkdownCellUnconnected extends React.Component {
             }}
```

Comment:
not going to block merge, but i think containerStyle is not needed anymore either

Reply:
yep already on it. note the WIP :)

Reply:
oh sorry thought this was ready to merge! looks good btw -- i was actually about to merge it...

Reply:
ping me when you're ready

Reply:
Almost there! Got side-tracked by the engagement survey